itests/dummydeal_test.go

```diff
@@ -50,6 +51,10 @@ import (
 
 var log = logging.Logger("boosttest")
 
+func init() {
+	build.MessageConfidence = 1
+}
```

Comment:
Does that have an effect... I was under the impression we need to adjust this on the actual devnet.